app/src/main/java/org/systers/mentorship/view/adapters/AchievementsAdapter.kt

```diff
@@ -0,0 +1,60 @@
+package org.systers.mentorship.view.adapters
+
+import android.support.v7.recyclerview.extensions.ListAdapter
+import android.support.v7.util.DiffUtil
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.achievement_list_item.view.*
+import org.systers.mentorship.R
+import org.systers.mentorship.models.Task
+
+
+/**
+ * This class is the recyclerview adapter for achievements
```

Comment:
this can be 2 lines instead of 4 lines of text right?

Reply:
I can bring it down to 3, because Android Studio by default wraps lines at 100 characters. Here's how it looks like now:

```
/**
 * This class is the recyclerview adapter for achievements. It is a subclass of [ListAdapter] for
 * easy async calculation of diffs using DiffUtil to provide nice animations when the data set
 * changes.
 */
```